target/linux/ath79/dts/ar9344_atheros_db120.dts

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Atheros DB120 reference board";
+	compatible = "atheros,db120", "qca,ar9344";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "db120:green:wlan2g";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wlan5g {
+			label = "db120:green:wlan5g";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		led_system: system {
+			label = "db120:green:system";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		usb {
+			label = "db120:green:usb";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port1>;
+			linux,default-trigger = "usbport";
+		};
+	};
+
+	leds-ath9k {
+		compatible = "gpio-leds";
+
+		wlan5g-ath {
+			label = "db120:green:wlan5g-ath";
+			gpios = <&ath9k 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
```

Comment:
Really phy1tpt like for wlan5g-nonath?

Reply:
This is the second atheros interface, hooked up via PCIe. Wasn't sure about the wlan5g-ath, but that's the best I could come up with.

Reply:
This is not about the name, but about using the very same "phy1tpt" trigger as for the other led?

Reply:
That's correct too, the board has 3 wireless-related LEDs:
 - 1 for the 2.4G via GPIO
 - 1 for the 5G via GPIO
 - 1 for the 5G via ath9k GPIO

So using phy1tpt for both 5G LEDs seems reasonable - being a reference board, Atheros would've wanted to expose everything.